SourceX/qol.cpp

```diff
@@ -237,4 +237,156 @@ void AutoGoldPickup(int pnum)
 	}
 }
 
+struct drawingQueue {
+	int itemID, row, col, x, y, width, height;
+	text_color color;
+	std::string text;
+	drawingQueue(int itemID, int row, int col, int x, int y, int width, int height, text_color color, std::string text)

```

Comment:
Doesn't look like you need this constructor. Remove it and construct with aggregate initialization instead: `drawingQueue { itemId, row, col, ... }`

Reply:
And why pass `std::string` by value instead of const reference?

Reply:
> Doesn't look like you need this constructor. Remove it and construct with aggregate initialization instead: `drawingQueue { itemId, row, col, ... }`

Thanks :) I didn't know about such a thing